go-update/src/test/java/org/reactome/release/goupdate/JavaVsPerlComparison.java

```diff
@@ -21,58 +26,98 @@
+
+		Map<String, Integer> diffNameToCount = new ConcurrentHashMap<>();
+		Map<String, Integer> sameNameToCount = new ConcurrentHashMap<>();
+
 		AtomicInteger diffCount = new AtomicInteger(0);
 		AtomicInteger sameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger regulationDiffCount = new AtomicInteger(0);
 		AtomicInteger regulationSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger peDiffCount = new AtomicInteger(0);
 		AtomicInteger peSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger catActDiffCount = new AtomicInteger(0);
 		AtomicInteger catActSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger accessionsMissingFromPerl = new AtomicInteger(0);
-		
+
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOBiologicalProcesses = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
+		Set<GKInstance> javaUpdatedGOBiologicalProcesses =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOMolecularFunctions = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_MolecularFunction);
+		Set<GKInstance> javaUpdatedGOMolecularFunctions =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_MolecularFunction);
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOCellularComponents = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_CellularComponent);
-		
+		Set<GKInstance> javaUpdatedGOCellularComponents =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_CellularComponent);
+
+		System.out.println("All Java GO instances obtained");
+
 		List<GKInstance> listOfAllJavaGOThings = new ArrayList<>();
 		listOfAllJavaGOThings.addAll(javaUpdatedGOCellularComponents);
 		listOfAllJavaGOThings.addAll(javaUpdatedGOMolecularFunctions);
 		listOfAllJavaGOThings.addAll(javaUpdatedGOBiologicalProcesses);
-		
+
+		@SuppressWarnings("unchecked")
+		Set<GKInstance> perlUpdatedGOBiologicalProcesses =
+			(Set<GKInstance>) perlUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
+		@SuppressWarnings("unchecked")
+		Set<GKInstance> perlUpdatedGOMolecularFunctions = (
+			Set<GKInstance>) perlUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_MolecularFunction);
+		@SuppressWarnings("unchecked")
+		Set<GKInstance> perlUpdatedGOCellularComponents =
+			(Set<GKInstance>) perlUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_CellularComponent);
+
+		System.out.println("All Perl GO instances obtained");
+
+		List<GKInstance> listOfAllPerlGOThings = new ArrayList<>();
+		listOfAllPerlGOThings.addAll(perlUpdatedGOCellularComponents);
+		listOfAllPerlGOThings.addAll(perlUpdatedGOMolecularFunctions);
+		listOfAllPerlGOThings.addAll(perlUpdatedGOBiologicalProcesses);
```

Comment:
Instead of `listOfAllPerlGoThings`, maybe it should become `listOfAllPerlGoInstances`?

Reply:
Changed: https://github.com/reactome/data-release-pipeline/pull/121/commits/ce8ce4883a21048b4e003618de3da7b8ba3ae538